test/integration/services/data/test-update-reduction.js

```diff
@@ -49,6 +52,16 @@ describe('/services/data/update-reduction', function () {
       })
   })
 
+  it('should update a reduction status and return an id ', function () {
+    return updateReductionStatus(addedReductionId, workloadOwnerId, reductionStatusType.ARCHIVED)
+      .then(function (result) {
+        // Store the id so that we can delete it after the test is complete
+        reductionResult.id = result
+        expect(result[0]).to.be.a('number')
+        expect(result).to.eql(addedReductionId)
```

Comment:
You could also retrieve the updated reduction from the DB and make sure the status has been updated

Reply:
Done